packages/wallet-specs/bin/uploadConfig.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+# Requirements: https://github.com/defunkt/gist
+# Usage:
+# > yarn uploadConfig $PROTOCOL
+#
+# Example
+# ❯ yarn uploadConfig direct-funding
+# yarn run v1.21.1
+# $ bin/uploadConfig.sh direct-funding
+# http://xstate.js.org/viz/?gist=7b60e4d910abec0a7b5368ac11f54ace
+
+PROTOCOL=$1
+FILENAME=src/protocols/${PROTOCOL}/protocol.config.js
```

Comment:
Is there a way we could generalize this so we could use it in `xstate-wallet` as well for workflows?

Currently the `xstate-wallet` as a `workflow` folder that contains a `.ts` file per a workflow (IE: `Workflows/application.ts`) but I'm open to reorganizing that so we can reuse this.

Maybe the easiest way for now is just to create a `uploadConfig.sh` for `xstate-wallet` and we can pull out shared logic later to `devtools`

Reply:
we could just accept the filename as the 1st argument. Then:
- in `wallet-protocols`, run `bin/uploadConfig.sh src/protocols/${YARNS_ARG}/protocol.config.js` for the `uploadConfig` script